src/coreclr/vm/codeman.cpp

```diff
@@ -909,6 +909,29 @@ BOOL IsFunctionFragment(TADDR baseAddress, PTR_RUNTIME_FUNCTION pFunctionEntry)
         pUnwindCodes += EpilogCount;
     }
 
+    return ((*pUnwindCodes & 0xFF) == 0xE5);
```

Comment:
this looks equivalent to `TARGET_LOONGARCH64` perhaps just keep it a single implementation?

Reply:
Registers and assembly instructions of RISC-V and LOONGARCH64 are more similar than ARM. I implemented RISC-V based on LOONGARCH64.

Reply:
hence wondering if they are the same can we remove duplication for now and just use the LoongArch implementation or is it expected to diverge soon?

Reply:
Sorry. I misunderstood your review. I updated it.
Thank you!